quality/errorprone/android.gradle

```diff
@@ -0,0 +1,9 @@
+apply plugin: 'net.ltgt.errorprone'
+
+dependencies {
+  annotationProcessor "com.uber.nullaway:nullaway:0.3.7"
```

Comment:
why not `0.4.0`?

Reply:
OK, latest was 0.3.7 before lunch. Will update.

Reply:
LOOOOL what a timing xD